administration/src/bp-modules/self-service/components/IconTextButton.tsx

```diff
@@ -7,7 +7,9 @@ export const IconTextButton = styled('button')`
   background-color: transparent;
   border: none;
   cursor: pointer;
-  font-size: 16px;
+  font-size: 14px;
   align-items: center;
   margin-bottom: 12px;
+  color: #595959;
```

Comment:
Please move this to a named constant

Reply:
i think it makes more sense to define a colors file at some point and define there everything. This is sth that should be created and used project wide.
I will create a ticket for that